### ISSUE TITLE
Variable nil vs. missing variable for field argument

### ISSUE THREAD

**Issue by hlship** [walmartlabs/lacinia#20]
From the GraphQL spec, concerning [null values](http://facebook.github.io/graphql/#sec-Null-Value)

```
The same two methods of representing the lack of a value are possible via variables by either providing the a variable value as null and not providing a variable value at all.
```

I'm pretty sure that the implementation will pass a nil for the argument value if the argument is bound to a variable that is simply not provided in the query.  If so, this must be changed.

**Comment by annapawlicka** [comment]
Looking into it.

**Comment by annapawlicka** [comment]
Resolved in https://github.com/walmartlabs/lacinia/pull/97

**Comment by annapawlicka** [comment]
Fixed with #102
